modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/account-menu/index.html

```diff
@@ -0,0 +1,74 @@
+<div class="account position-relative" tabindex="9">
+<button class="account-info align-items-center d-flex ${themeDisplay.isSignedIn()?then('py-1 pl-2', '')}" data-toggle="liferay-dropdown" id="account-menu-id">
```

Comment:
Some of the indents are off in this file. This first `<button>` does not line up with the `</button>` tag. There are some other indents that don't line up in this file too, can you find them?

Make sure the indenting is accurate in files to have better readability for everyone working on it in the future